stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/Android.java

```diff
@@ -109,4 +115,33 @@ public void bindingWithViewModel() {
             });
         }
     }
+
+    /**
+     * @see <a href="https://getstream.io/nessy/docs/chat_docs/android_chat_ux/attachmentgallery">Attachment Gallery Activity</a>
```

Comment:
I think it's not an appropriate link for users. It's only for internal stream usage and editing in CMS